tasks/pre_checks/001_validate_network_interfaces.yml

```diff
@@ -70,6 +70,37 @@
         reject('skipped') | map(attribute='bond_item.ansible_facts.otopi_net_host') | list }}
     register: otopi_host_net
   - debug: var=otopi_host_net
+  - name: Collect interfaces type
+    shell: nmcli con show {{ nic }} | grep connection.type
+    with_items:
+      - "{{ otopi_host_net }}"
+    loop_control:
+      loop_var: nic
+    register: interfaces_type
+  - debug: var=interfaces_type
+  - name: Check if there are team devices
+    set_fact:
+      is_team: "{{ nic_if.stdout.find('team') > 0 }}"
+    when: nic_if.stdout.find('team') != -1
+    with_items:
+      - "{{ interfaces_type.results }}"
+    loop_control:
+      loop_var: nic_if
+    register: team_list
+  - debug: var=team_list
+  - name: Get list of team devices
+    set_fact:
+      team_if: "{{  team_list.results | reject('skipped') | map(attribute='nic_if.nic') | list }}"
+  - debug: var=team_if
+  - name: Filter unsupported interfaces type
+    set_fact:
+      otopi_host_net: "{{ otopi_host_net | difference(team_if) }}"
+    register: otopi_host_net
```

Comment:
Please don't re-use variables. 
it's confusing to debug a playbook with variables that repeat themselves with different values.
Additionally, `otopi_` prefix is for Otopi to know what values to use after the playbook finishes. Here it might get the first list rather than the second.

Use another variable name that is not `otopi_host_net` here or above.